tests/test_market.py

```diff
@@ -132,6 +133,59 @@ def market_matcher():
     return PayAsBidMatcher()
 
 
+class TestTwoSidedMarket:
+    """Class Responsible for testing two sided market's functionality.
+
+    TODO: bring all related two sided market tests from this module into this class.
+    """
+    def test_double_sided_validate_requirements_satisfied(self, market):
+        offer = Offer("id", now(), 2, 2, 'other', 2,
```

Comment:
Nit:
```suggestion
        offer = Offer("id", now(), 2, 2, "other", 2,
```

Reply:
The same applies to the rest of the PR (make sure to run the linter to catch these)